Apps/Clouseau/clouseau.asd

```diff
@@ -0,0 +1,81 @@
+;;;; Copyright (C) 2018, 2019 Jan Moringen
+;;;;
+;;;; This library is free software; you can redistribute it and/or
+;;;; modify it under the terms of the GNU Library General Public
+;;;; License as published by the Free Software Foundation; either
+;;;; version 2 of the License, or (at your option) any later version.
+;;;;
+;;;; This library is distributed in the hope that it will be useful,
+;;;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;;;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+;;;; Library General Public License for more details.
+;;;;
+;;;; You should have received a copy of the GNU Library General Public
+;;;; License along with this library; if not, write to the
+;;;; Free Software Foundation, Inc., 59 Temple Place - Suite 330,
+;;;; Boston, MA  02111-1307  USA.
+
+;;; This is a complete rewrite which does not share any code or
+;;; architecture with the old inspector also called "clouseau". It
+;;; keeps the system and package name since it should be (and in order
+;;; to make it) usable as a drop-in replacement.
+(defsystem "clouseau"
+  :description "A graphical inspector for arbitrary Common Lisp objects."
+  :license     "LGPL-2.1+"
+  :author      "Jan Moringen <jmoringe@techfak.uni-bielefeld.de>"
+
+  :version     (:read-file-form "version-string.sexp")
```

Comment:
Say what?

Reply:
I'd rather put a version number in asd definition but I don't  have a good argument except that it won't work on asdf2. So that was a dissatisfied mumbling about asdf making backward incompatible changes every now and then ,)